api/src/certification/shared/infrastructure/repositories/certification-course-repository.js

```diff
@@ -100,6 +101,13 @@ async function get({ id }) {
 
     certificationCourseDTO.numberOfChallenges =
       configuration?.maximumAssessmentLength ?? config.v3Certification.numberOfChallengesPerCourse;
+
+    candidateDTO = await knexConn('certification-candidates')
```

Comment:
Selectionner uniquement l'utile (reduction de la charge BDD)

```suggestion
    candidateDTO = await knexConn('certification-candidates').select('accessibilityAdjustmentNeeded')
```